### DISCUSSION TITLE
📦  Kor at OPT-NC and packaging tracking usecase

### DISCUSSION THREAD

**Discussion by adriens** [eyurtsev/kor#210] [Show and tell]
Hi guys, just a kind & quick feedback to let you know there are people here at [opt-nc](https://github.com/opt-nc/) talking about Kor:

- [🎦 YT Video](https://youtu.be/BN5Fdtvu21Q?si=kYf6FQzfbLAP2qrI&t=1567)
- DEV.to article : [🦜🔗 Track Package Delivery w. Langchain Agent Custom Tools](https://dev.to/adriens/track-package-delivery-w-langchain-agent-custom-tools-18jf)

Hopefully you'll like it. I have a lot of use cases for Kor, and for sure I'll stick on this project for data quality centric issues.

Thanks a lot for the great job 👏 ❣️

**Comment by eyurtsev** [comment]
Thanks for letting us know @adriens :heart:  :)

**Reply by adriens** [reply]
Thanks for the great library, there are plenty of use cases around here 🤩
